pkgs/os-specific/linux/busybox/default.script

```diff
@@ -0,0 +1,66 @@
+#!@bash@/bin/sh
```

Comment:
I'm wondering where this script came from, and if we can get it from an upstream source instead?

Reply:
It came from debian. For instance http://deb.debian.org/debian/pool/main/b/busybox/busybox_1.30.1-4.debian.tar.xz

The only changes I've made from upstream is that I have replaced all references to `some-tool` with `@some-pkgs@/bin/some-tool` and I have removed a few places that depended on `/sbin/resolvconf`.

I can rewrite this to use the upstream `default.script` instead.

I am unsure what to do about the `/sbin/resolvconf` part. It obviously will not work on NixOS, however I am not really sure if the typical user of this binary will care about that anyways?

Reply:
Given how many patches need to applied to this it might ok to just leave this version here.
It might be good to document where it came from though.

Reply:
I've updated the PR to get the script from a debian tarball instead. The newer version needs fewer patches as well.

Reply:
If you like this version better, I can do a rebase to get rid of the other one.